web/templates/settings/data_exports_admin.hbs

```diff
@@ -44,7 +44,7 @@
                 <th>{{t "Status" }}</th>
                 <th class="actions">{{t "Actions" }}</th>
             </thead>
-            <tbody id="admin_exports_table" class="required-text" data-empty="{{t 'No exports.' }}"></tbody>
+            <tbody id="admin_exports_table" class="required-empty-text" data-empty="{{t 'No exports found.' }}"></tbody>
```

Comment:
Changing `required-text` to `required-empty-text` should be done in a separate commit. And also these message changes, where there is no search element, should be done in separate commit.